src/test/unit_tests/jest_tests/test_nc_upgrade_manager.test.js

```diff
@@ -323,6 +321,17 @@ describe('nc upgrade manager - upgrade config directory', () => {
             await expect(nc_upgrade_manager_higher_version._verify_config_dir_upgrade(system_data, expected_version, [hostname]))
                 .rejects.toThrow(expected_err_msg);
         });
+
+        it('fail on already in progress upgrade', async () => {
+            const expected_version = pkg.version;
+            const system_data = {
+                [hostname]: { current_version: pkg.version },
+                config_directory: { in_progress_upgrade: { from_version: 'mock-version' } }
+            };
+            const expected_err_msg = `config dir upgrade can not be started - there is already an ongoing upgrade system_data.config_directory.in_progess_upgrade=${util.inspect(system_data.config_directory.in_progress_upgrade)}`;
```

Comment:
(For the future) 2 things that can be easier when writing the tests:
1. Move the error messages nc_upgrade_manager.js` to const that we can reuse it
2. Check partial match of the message (would create less "strict" test).